docs/track.rst

```diff
@@ -447,6 +447,7 @@ The ``schedule`` element contains a list of tasks that are executed by Rally, i.
 * ``clients`` (optional, defaults to 1): The number of clients that should execute a task concurrently.
 * ``warmup-iterations`` (optional, defaults to 0): Number of iterations that each client should execute to warmup the benchmark candidate. Warmup iterations will not show up in the measurement results.
 * ``iterations`` (optional, defaults to 1): Number of measurement iterations that each client executes. The command line report will automatically adjust the percentile numbers based on this number (i.e. if you just run 5 iterations you will not get a 99.9th percentile because we need at least 1000 iterations to determine this value precisely).
+* ``ramp-up-time-period`` (optional, defaults to 0): Rally will start clients gradually. It reaches the number specified by ``clients`` at the end of the specified time period. See the section on :ref:`ramp-up <track_ramp_up>` for more details.
```

Comment:
presumably this value is in seconds?

Reply:
Good point. Addressed in https://github.com/elastic/rally/pull/1266/commits/5232776893b4f2ced328f06c3f2235c2c20ccf9b.